frontend/src/components/MessagingDisplay.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import '../styles/messaging.scss';
+
+function MessagingDisplay(props){
+
+    const containerSize = props.largeDisplay != "true" ? "message-display-container-small" :
```

Comment:
Can we please switch the condition? To `props.largeDisplay === "true"`.. again, easier on the brain